test/server/routes/putScrumRoute.test.js

```diff
@@ -0,0 +1,46 @@
+const test = require('tape');
+const request = require('supertest');
+const connect = require('../../../server/database/config/connection');
+const app = require('../../../server/app');
+// query for dynamic test
+const selectOneUser = () => connect.query('select id from scrums LIMIT 1');
+
+test('put in /api/v1/scrums/:scrumId (with valid data)', (t) => {
+  selectOneUser()
+    .then((res) => {
+      if (res.rowCount !== 0) {
+        console.log(res.rows[0].id.toString());
+        return res.rows[0].id;
+      }
+      t.equal(res.rowCount === 0, true, 'there is no project in data base');
```

Comment:
I think we talked about this before, it doesn't make sense to put this test here, if you want to test failer case, you should check it in a separate test with a proper title